tests/migration/deregistration_system.pm

```diff
@@ -0,0 +1,18 @@
+# Copyright 2017 SUSE LLC
```

Comment:
New module file, it should be 2023, so for other new module files.

Reply:
yes, updated.